### PR TITLE
evaluateGraphQLResolver support non null fields

### PR DESCRIPTION
**Pull request by michalsanger**

### CODE REVIEW

File: src/test-utils/package.json [hunk 1]
```diff
@@ -4,14 +4,12 @@
...
+  "version": "0.4.1",
   "main": "src/index",
   "sideEffects": false,
   "dependencies": {
     "@adeira/js": "^1.2.0",
-    "@babel/runtime": "^7.9.2"
-  },
-  "devDependencies": {
+    "@babel/runtime": "^7.9.2",
     "graphql": "^14.6.0"
```
Comment (mrtnzlml) [inline src/test-utils/package.json:13]: Why is this necessary? 🤔 Shouldn't it be a `peerDependency`?

File: src/test-utils/src/evaluateGraphQLResolver.js [hunk 2]
```diff
@@ -18,17 +20,30 @@ export default function evaluateGraphQLResolver(
...
+    });
+  }
+  return defaultFieldResolver(
+    testValue,
+    argsValue ?? {},
+    contextValue,
+    ({
+      fieldName: field.name,
+      path: { key: 'mocked' },
+    }: any),
```
Comment (mrtnzlml) [inline src/test-utils/src/evaluateGraphQLResolver.js:47]: Would it be possible to sync these params with the direct `field.resolve` on line 36?

File: src/test-utils/package.json (hunk 1 above)
Comment (michalsanger) [reply src/test-utils/package.json:15]: fixed

File: src/test-utils/src/evaluateGraphQLResolver.js (hunk 2 above)
Comment (michalsanger) [reply src/test-utils/src/evaluateGraphQLResolver.js]: I tried
